rfq/order.go

```diff
@@ -284,6 +285,8 @@ func (c *AssetSalePolicy) GenerateInterceptorResponse(
 		fn.None[[]rfqmsg.ID](),
 	)
 
+	htlcRecord.SetNoopAdd(rfqmsg.UseNoOpHTLCs)
```

Comment:
Can add a comment here that: we set this to ensure that when we send on the outgoing link, we don't push any sats over if we don't need to.